plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/exporter/DataExporterCSV.java

```diff
@@ -101,6 +103,10 @@ public void init(IStreamDataExporterSite site) throws DBException
         if (ROW_DELIMITER_DEFAULT.equalsIgnoreCase(this.rowDelimiter.trim())) {
             this.rowDelimiter = GeneralUtils.getDefaultLineSeparator();
         }
+        this.lineFeedEscapeString = CommonUtils.toString(properties.get(PROP_LINE_FEED_ESCAPE_STRING), "")
+            .replace("\\t", "\t")
+            .replace("\\n", "\n")
+            .replace("\\r", "\r");
```

Comment:
Maybe, just maybe, we want to allow users to use, e.g., literal `\t` by doubling the slash `\\t`?

I have the following regex that does that, but I don't know of an easy way to convert it to a symbol those escapes correspond to: `(?<!\\)\\[rnt]` (matches `\r`, `\n`, and `\t`)